lib/src/coap_config.dart

```diff
@@ -98,4 +83,12 @@ abstract class DefaultCoapConfig {
 
   /// Can be used to specify the Ciphers that should be used by OpenSSL.
   String? get dtlsCiphers => null;
+
+  /// Custom libssl instance that can be registered if OpenSSL
+  /// should not be available at the default locations.
+  OpenSsl? get libSslInstance => null;
+
+  /// Custom libcrypto instance that can be registered if OpenSSL
+  /// should not be available at the default locations.
+  OpenSsl? get libCryptoInstance => null;
```

Comment:
I am wondering if the type of these should rather be changed to `DynamicLibrary`. This would also get rid of the `OpenSsl` reexport.

Reply:
Changed this now in the latest version of dtls2.